### PR TITLE
Add -C flag to knife winrm.

### PR DESCRIPTION
**Pull request by tduffield**
Depends on the release of chef/knife-windows#417.

### CODE REVIEW

File: chef_master/source/windows.rst [hunk 1]
```diff
@@ -1991,6 +1991,10 @@ This argument has the following options:
 ``-f CA_TRUST_FILE``, ``--ca-trust-file CA_TRUST_FILE``
    Optional. The certificate authority (CA) trust file used for SSL transport.
 
+``-C NUM``, ``--concurrency NUM``
+   Changed in knife-windows 1.9.0; included in Chef DK 1.2.20.
```
Comment (tduffield) [inline chef_master/source/windows.rst]: This has not been included in  ChefDK release yet.

**Review by davidwrede — APPROVED** [review]
👍

**Comment by tduffield** [comment]
knife 1.9.0 has been released. This should be good to merge.

**Comment by kagarmoe** [comment]
gif-keyboard-973940458679382102

**Pull request merged**